operatorcourier/api.py

```diff
@@ -60,6 +60,11 @@ def build_verify_and_push(namespace, repository, revision, token, source_dir=Non
     bundle = build_and_verify(source_dir, yamls)
 
     if bundle is not None:
+        # Set data fields to strings to match expected bundle format
+        bundle['data']['customResourceDefinitions'] = yaml.dump(bundle['data']['customResourceDefinitions'])
+        bundle['data']['clusterServiceVersions'] = yaml.dump(bundle['data']['clusterServiceVersions'])
+        bundle['data']['packages'] = yaml.dump(bundle['data']['packages']
```

Comment:
Can you actually move this to the end of the build_and_verify() method so that method returns the correctly formatted bundle? I think it has to be after the validation so the fields can be referenced directly.